### PR TITLE
[stable/sonatype-nexus] Nexus update version to 3.11.0

### PR DESCRIPTION
**Pull request by bbriggs**
**What this PR does / why we need it**: Updates to latest version of `quay.io/travelaudience/docker-nexus` container at time of PR

**Which issue this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*: fixes #

**Special notes for your reviewer**: [Current list of tags for image](https://quay.io/repository/travelaudience/docker-nexus?tag=latest&tab=tags)

### CODE REVIEW

File: stable/sonatype-nexus/Chart.yaml [hunk 1]
```diff
@@ -1,5 +1,5 @@
 name: sonatype-nexus
-version: 1.2.0
+version: 1.2.1
```
Comment (rjkernick) [inline stable/sonatype-nexus/Chart.yaml]: Please bump the minor version instead

**Review by rjkernick — CHANGES_REQUESTED** [review]

File: stable/sonatype-nexus/Chart.yaml (hunk 1 above)
Comment (bbriggs) [reply stable/sonatype-nexus/Chart.yaml]: Sure thing.

**Comment by bbriggs** [comment]
Current app version, according to their repo: https://github.com/sonatype/nexus-public/releases/tag/release-3.11.0-01

**Pull request merged**